packages/widget_toolkit/bin/build_runner_build.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env sh
 
-flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
+flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs &&\
+rm lib/src/lib_item_picker/blocs/item_picker_bloc.rxb.g.dart &&\
```

Comment:
Why those files need to be removed ?

Reply:
Because we have them modified with "rxb_g.dart" sufix. The reason is that generator does not support generics. Every time we run the generator, we need to delete the generated versions, since they decrease our pub score